frontend/src/components/Banner.tsx

```diff
@@ -18,6 +18,12 @@ export type BannerProps = {
     onClick?: () => void;
     gaViewPing?: Parameters<typeof useGaViewPing>[0];
```

Comment:
Haha, so funny that the regular CTA wasn't big enough :P 

The current approach is fine, but since it's interesting to see how different people approach problems, I'll share what my intuition was :) SInce large CTA is otherwise the same as the regular CTA, I'd just have added an optional property to that definition, and then use that to determine which class to add:

```suggestion
    gaViewPing?: Parameters<typeof useGaViewPing>[0];
    size?: "regular" | "large";
```

(i.e. that adds an optional `size` property that can be set to `"regular"` or `"large"`, and then you could do something like `className={props.cta.size === "large" ? styles[["cta-large-button"] : styles.cta }`)

Again, no need to change this, just sharing in case it's interesting :)

Reply:
Oh actually, it also affects its position in the DOM. Well, never mind then - though I'm leaving this in because the thought process might still be interesting :P

Reply:
Oh that is awesome to know in the future - I figured there was some kind of way to do it simply like that. Would I have had to do add the size property for all the other banners in ProfileBanners.tsx if that had been the case, or just the one?

Reply:
Nope, the `?` marks it as optional (i.e. `size?: "regular" | "large"` is roughly the same as `size: "regular" | "large" | undefined`)!